client/components/Intro/Intro.jsx

```diff
@@ -5,17 +5,8 @@ import Paper from '@material-ui/core/Paper';
 import Typography from '@material-ui/core/Typography';
 import Divider from '@material-ui/core/Divider';
 
-// import './intro.scss';
-
-
-// Image import here:
-// import Image from '../Intro/Testimage.jpg';
-// import Image from '.components/Intro/Testimage.jpg';
-
-
-// NOTES
-// Decide whether to keep the card material-ui component design
-// Would like to add an image representing "coding resources"
+import './Intro.scss';
+import image from './image.jpg';
```

Comment:
So, I guess my problem was adding the image in the component folder, when it should be on the main root.